flows/prefect2_only/deployment_run.py

```diff
@@ -45,25 +44,28 @@ async def read_flow_run(flow_run_id):
 
 
 def main():
-    # Create deployment
-    deployment = Deployment.build_from_flow(flow=hello, name="test-deployment")
-    deployment_id = deployment.apply()
-
-    # Create a flow run
-    flow_run = anyio.run(create_flow_run, deployment_id)
-
-    env = os.environ.copy()
-    env["PREFECT__FLOW_RUN_ID"] = str(flow_run.id)
-    subprocess.check_call(
-        [sys.executable, "-m", "prefect.engine"],
-        env=env,
-        timeout=30,
-        stdout=sys.stdout,
-        stderr=sys.stderr,
-    )
-
-    flow_run = anyio.run(read_flow_run, flow_run.id)
-    assert flow_run.state.is_completed(), flow_run.state
+    if Version(prefect.__version__) <= Version("2.19.0"):
+        from prefect.deployments import Deployment
```

Comment:
This moves `Deployment` import behind a version check as we won't have `Deployment` after 2.19.0. No other changes